### DISCUSSION TITLE
How to launch silently on startup in Linux?

### DISCUSSION THREAD

**Discussion by yllekz** [debauchee/barrier#1565] [Q&A]
Hi there, I was curious, is there a way to silently start the Barrier server on Linux without getting the GUI window popping up on startup? I'm using ``barrier`` as the startup command but can't figure out how to get rid of the initial window from popping up. I checked the Wiki but nothing stuck out to me.

**Comment by jdm0830** [comment]
maybe this? https://stackoverflow.com/questions/18881221/how-to-start-a-batch-file-minimized-with-task-scheduler-in-windows-8-comspec/27701957

**Reply by yllekz** [reply]
This would be Linux, not Windows..

**Comment by juanibiapina** [comment]
There seems to be a barriers command which could be the solution.
